readme.md

```diff
@@ -1 +1,54 @@
-readme
\ No newline at end of file
+**F**irefo**x** **A**ccounts **P**age **O**bject **M**odel
+===============================================
+Webdriver compatible page object model and utilities for Firefox Accounts
+
+Overview
+-------------
+This package contains a utility to create a test Firefox Account on either the dev or prod instance of Firefox Accounts,
+as well as a set of page objects that can be used to interact with Firefox Accounts' sign in screens.
+
+Usage
+-----
+To create a test Firefox Account, use the `create_account` method in the `FxATestAccount` object.
+You should pass the base url for the site for which you are creating the account into the constructor
+for `FxATestAccount`, or, if you know you want to create a production Account, you can omit that argument.
+
+Example:
+```
+from fxapom.fxapom import FxATestAccount
+acct = FxATestAccount(base_url='https://www-dev.allizom.org').create_account()
+```
+
+To sign in via Firefox Accounts, use the `sign_in` method in the `WebDriverFxA` object,
+passing in the email addresss and password.
+
+Example:
+```
+from fxapom.fxapom import WebDriverFxA
+fxa = WebDriverFxA(mozwebqa)
+fxa.sign_in(email_address, password)
+```
+
+To create an account and then use it to sign in, use both tools described above.
+
+Example:
+```
+from fxapom.fxapom import FxATestAccount
+from fxapom.fxapom import WebDriverFxA
+acct = FxATestAccount(base_url='https://www-dev.allizom.org').create_account()
+fxa = WebDriverFxA(mozwebqa)
+fxa.sign_in(acct.email, acct.password)
+```
+
+Running The Tests
+-----------------
+* Install the requirements using `pip install -r requirements.txt`
```

Comment:
Yeah, that error above, @AndreiH, wouldn't seem to be related to running `setup`. I'm not sure what the issue is you are seeing.